modAionImpl/src/org/aion/zero/impl/config/dynamic/InFlightConfigChangeException.java

```diff
@@ -0,0 +1,7 @@
+package org.aion.zero.impl.config.dynamic;
+
+public class InFlightConfigChangeException extends Exception {
```

Comment:
I generally like specific exceptions, but I'll see if it can be simplified.

Reply:
So, right now I have InFlightConfigChangeException and then two subclsases InFlightConfigChangeNotAllowedException and RollbackException.  

I think I can get rid of InFlightConfigChangeNotAllowedException and just use InFlightConfigChangeException all the time.

RollbackException probably needs to stay because the logic is if a config change failed, do a rollback, but if the rollback fails, don't try to rollback again (avoid endless rolling back loop).  It should also probably not subclass from InFlightConfigChangeException